config.json

```diff
@@ -772,13 +784,13 @@
           "type": "autocomplete",
           "autocompleteType": "function",
           "functionName": "listProjectsAuto",
-          "description": "GCP Project defines how your app interacts with services and resources",
+          "description": "Google Cloud project forms the basis for creating, enabling, and using all Google Cloud services including managing APIs and managing permissions for Google Cloud resources",
           "placeholder": "select using autocomplete",
           "learnUrl": "https://cloud.google.com/resource-manager/docs/creating-managing-projects"
         },
         {
           "name": "network",
-          "viewName": "VPC Network Name *",
```

Comment:
and this one?

Reply:
No. It is useful for narrowing the autocomplete search for subnets, but left blank the method still works.